sources/engine/Stride.Assets/StridePackageUpgrader.cs

```diff
@@ -251,15 +251,15 @@ public override bool UpgradeBeforeAssembliesLoaded(PackageLoadParameters loadPar
                         if (tfm != null)
                         {
                             // Library
-                            if (tfm.EvaluatedValue.StartsWith("net6", StringComparison.Ordinal) && solutionProject.Type == ProjectType.Library)
+                            if (tfm.EvaluatedValue.StartsWith("net8", StringComparison.Ordinal) && solutionProject.Type == ProjectType.Library)
                             {
                                 // In case it's a single TargetFramework, add the "s" at the end
                                 tfm.Xml.Name = "TargetFrameworks";
                                 tfm.Xml.Value = "net8.0";
                                 isProjectDirty = true;
                             }
                             // Executable
-                            else if ((tfm.EvaluatedValue.StartsWith("net6", StringComparison.Ordinal)) && solutionProject.Type == ProjectType.Executable)
+                            else if ((tfm.EvaluatedValue.StartsWith("net8", StringComparison.Ordinal)) && solutionProject.Type == ProjectType.Executable)
```

Comment:
This should be kept as is for proper incremental upgrade.

Reply:
Sorry, I read the code too quick 🤦. Undone